packages/core/prompt/BasePromptSession.ts

```diff
@@ -369,6 +372,22 @@ export abstract class BasePromptSession {
 			break;
 		case "Complete & Run":
 			const config = ProjectConfig.localConfig();
+
+			if (config.project.framework === "angular" && !config.packagesInstalled) {
+				// TODO: should we add check if there are paid components at all
+				const usePaidAngular = await this.getUserInput({
+					type: "list",
+					name: "usePaidAngular",
+					message: "Do you have paid subscription:",
```

Comment:
Nit pick, but the name variable should be `useLicensedAngular` IMO.
Also, minor typo
```suggestion
					message: "Do you have a paid subscription:",
```

Reply:
we should revise very carefully all the messages in this PR. The existing messages are just a place holders :)